templates/config.tpl

```diff
@@ -136,6 +136,12 @@ allowedHTMLLoadExtensions[] = {
 allowedFilePatching = <%= allowedFilePatching %>;
 <% } %>
 
+<% if (debug !== null) { %>
+// More verbose logging (e.g. stack traces)
+// set to 1 to activate
+debug = <%= debug %>;
```

Comment:
Is this documented somewhere? I can only find the [startup argument](https://community.bistudio.com/wiki/Arma_3:_Startup_Parameters#Developer_Options) on the Biki

No harm, just curious :)

Reply:
yes I think that's where I got it from.

Reply:
The startup argument is added in `src/server.js`, does this flag in server.cfg also enable additional debugging?